.gitlab-ci.yml

```diff
@@ -19,26 +18,42 @@ variables:
 cache:
   paths:
     - .cache/pip
-    - venv/
 
-before_script:
-  - python -V  # Print out python version for debugging
-  - pip install -r requirements.txt
-  - pip install python-coveralls
-
-test:
+.tests:
   stage: test
   script:
-    - echo $TEST_SUITE
-    - echo $TEST_SUITE > test.sh
-    - bash test.sh
+    - python -V  # Print out python version for debugging
+    - pip install -r requirements.txt
+    - pip install python-coveralls
+    - python -m coverage run -m pytest
     - coverage report
     # Test that it can be started using the checked-out source code:
     - python -m pyega3.pyega3 --help
     # Test that it can be started using the installed package:
     - pip install -e . && pyega3 --help
   #  - coveralls
 
+test_python36:
+  extends: .tests
+  image: python:3.6
```

Comment:
I couldn't use python:3.6-slim, because slim doesn't contain some libraries which are needed by the psutil library.